src/itwinai/loggers.py

```diff
@@ -349,11 +349,11 @@ class ConsoleLogger(Logger):
 
     def __init__(
         self,
-        savedir: str = "mllogs",
+        savedir: Path = "mllogs",
         log_freq: Union[int, Literal["epoch", "batch"]] = "epoch",
         log_on_workers: Union[int, List[int]] = 0,
     ) -> None:
-        savedir = savedir = Path(savedir) / "simple-logger"
+        savedir = Path(savedir) / "simple-logger"
```

Comment:
Same here, is `savedir / "simple_logger"` sufficient?

Reply:
here it could also be a string. Fixing type hint